driver-service/src/main/java/com/modsen/driverservice/mappers/AutoMapper.java

```diff
@@ -0,0 +1,24 @@
+package com.modsen.driverservice.mappers;
+
+import com.modsen.driverservice.dto.AutoDto;
+import com.modsen.driverservice.entities.Auto;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AutoMapper {
+    private final ModelMapper mapper;
+
+    public AutoMapper(){
+        this.mapper = new ModelMapper();
```

Comment:
Вынеси создание new ModelMapper() в пакет конфиг в отдельный конфигурационный класс, в котором будет создаваться бин ModelMapper
```suggestion
       @Configuration
public class ModelMapperConfig {
    @Bean
    public ModelMapper modelMapper() {
        return new ModelMapper();
    }
}
```